src/msg/msg_types.h

```diff
@@ -384,6 +385,24 @@ struct entity_addr_t {
     }
   }
 
+  std::string ip_only_to_str() {
```

Comment:
please mark this method `const`.

Reply:
At first I also considered this problem, but found that all the implementation in "msg_types.h"

Reply:
i don't get you. could you rephrase you reply?

Reply:
sorry, I may not have described it clearly, I mean whether all the implementations in msg_types.h are placed in msg_types.cc？

Reply:
not all.

Reply:
let's put this one in the .cc file.  we can move others later.

Reply:
@liewegas 
ok! updated it